doc/unordered/changes.adoc

```diff
@@ -6,6 +6,11 @@
 :github-pr-url: https://github.com/boostorg/unordered/pull
 :cpp: C++
 
+== Release 1.85.0
+
+* Optimized `emplace()` for a `value_type` or `init_type` (if applicable) argument to only allocate once.
```

Comment:
The use of "allocate once" is confusing to me here. What's meant by it?

Reply:
I'll make it more clear. When we're given a `value_type` or `init_type` as an argument to `emplace()`, we can bypass creating an intermediate value, which we would otherwise create.